virgil/friend/friend.py

```diff
@@ -314,8 +321,9 @@ def on_message(self, message, verbose: bool = False):
 @click.option("--token", default=None, help="The token for the discord bot.")
 @click.option("--backend", default="gemma", help="The backend to use for the chat.")
 @click.option("--max-history-length", default=25, help="The maximum length of the chat history.")
-def main(token, backend, max_history_length):
-    bot = Friend(token=token, backend=backend, max_history_length=max_history_length)
+@click.option("--prompt_filename", default="prompt.txt", help="The filename for the prompt.")
```

Comment:
nit: `--prompt-filename` would be more consistent. I'd probably just leave it at `--prompt`

Reply:
will change to just --prompt

Reply:
also, not clear that the file is relative to "virgil/friend" instead of the working directory

Reply:
one